modules/spring-boot-autoconfigure/README.txt

```diff
@@ -0,0 +1,45 @@
+Apache Ignite Spring Boot Autoconfigure Module
+----------------------------------------
+
+Apache Ignite Spring Boot Autoconfigure module provides autoconfiguration capabilities for Spring-boot based applications.
+
+Features
+---------------------------
+This module provides the ability to integrate `Ignite` into you spring-boot application with zero(or minimal) configuration.
```

Comment:
can you please add a minimum config file in resources or add sample configuration details in the README for reference

https://github.com/nizhikov/ignite-spring-boot-autoconfigure-example/blob/master/src/main/resources/application-node.yml

Reply:
1. Examples added to both modules.
2. There is no required configuration for this module. If no additional properties provided - `Ignite` node with the default configuration will be started and added to the `ApplicationContext`.